Author: Takashi Iwai <tiwai@suse.de>
Date: Tue, 24 May 2005 11:16:56 +0000 (+0200)
Source: http://www.kernel.org/git/gitweb.cgi?p=linux/kernel/git/perex/alsa.git;a=commitdiff;h=b9cce37411edd668ae5dbc16aa9653800c8e0744

  [ALSA] Add Shuttle SN25P support
  
  ICE1712 driver
  Added the support for Shuttle SN25P.
  
  Signed-off-by: Takashi Iwai <tiwai@suse.de>

--- a/sound/pci/ice1712/vt1720_mobo.c
+++ b/sound/pci/ice1712/vt1720_mobo.c
@@ -110,6 +110,15 @@ struct snd_ice1712_card_info snd_vt1720_
 		.eeprom_size = sizeof(k8x800_eeprom),
 		.eeprom_data = k8x800_eeprom,
 	},
+	{
+		.subvendor = VT1720_SUBDEVICE_SN25P,
+		.name = "Shuttle SN25P",
+		/* identical with k8x800 */
+		.chip_init = k8x800_init,
+		.build_controls = k8x800_add_controls,
+		.eeprom_size = sizeof(k8x800_eeprom),
+		.eeprom_data = k8x800_eeprom,
+	},
 	{ } /* terminator */
 };
 
--- a/sound/pci/ice1712/vt1720_mobo.h
+++ b/sound/pci/ice1712/vt1720_mobo.h
@@ -33,6 +33,7 @@
 #define VT1720_SUBDEVICE_ZNF3_150	0x0f2741f6
 #define VT1720_SUBDEVICE_ZNF3_250	0x0f2745f6
 #define VT1720_SUBDEVICE_9CJS		0x0f272327
+#define VT1720_SUBDEVICE_SN25P		0x97123650
 
 extern struct snd_ice1712_card_info  snd_vt1720_mobo_cards[];
 
